### PR TITLE
ci: run packit builds for commits on master and releases

### PR DESCRIPTION
**Pull request by t-8ch**
Cc @mrc0mmand

### CODE REVIEW

**Comment by t-8ch** [comment]
Maybe we also want to run the tests as root to have more coverage?

**Comment by mrc0mmand** [comment]
> Maybe we also want to run the tests as root to have more coverage?

That's unfortunately not possible in Copr (the backend Packit uses), I already played around with this in the systemd repo to no avail (it's even mentioned somewhere in the Copr docs).

File: .packit.yaml [hunk 1]
```diff
@@ -50,3 +50,20 @@ jobs:
       - fedora-rawhide-ppc64le
       - fedora-rawhide-s390x
       - fedora-rawhide-x86_64
+  - job: copr_build
```
Comment (mrc0mmand) [inline .packit.yaml:53]: You can make this slightly less verbose using the *YAML merge key syntax* which Packit supports - https://packit.dev/docs/configuration/#packit-service-jobs

Comment (t-8ch) [reply .packit.yaml]: Good point, done.

**Pull request merged**